### PR TITLE
Add notebook output

### PR DESCRIPTION
**Pull request by cfkanesan**

### CODE REVIEW

File: notebooks/3_Retrieve_Echotop_and_Regrid.ipynb [hunk 1]
```diff
@@ -13,20 +13,20 @@
...
    "metadata": {},
    "outputs": [],
    "source": [
     "import os\n",
     "\n",
     "os.environ[\"POLYTOPE_ADDRESS\"] = \"https://polytope-dev.mchml.cscs.ch\"\n",
-    "#os.environ[\"POLYTOPE_USERNAME\"] = \"admin\"\n",
-    "#os.environ[\"POLYTOPE_PASSWORD\"] = \"xxxxxxxxxxxxx\"\n"
+    "os.environ[\"POLYTOPE_USERNAME\"] = \"admin\"\n",
+    "os.environ[\"POLYTOPE_PASSWORD\"] = \"jxcmecNjioyMitMk\"\n"
```
Comment (ninaburg) [inline notebooks/3_Retrieve_Echotop_and_Regrid.ipynb:29]:
```suggestion
    "os.environ[\"POLYTOPE_PASSWORD\"] = \"xxxxxxxxxxxxx\"\n"
```

File: notebooks/4_Location_and_TimeSeries_Access.ipynb [hunk 2]
```diff
@@ -28,20 +30,21 @@
     "\n",
     "\n",
     "os.environ[\"POLYTOPE_USERNAME\"] = \"admin\"\n",
-    "os.environ[\"POLYTOPE_PASSWORD\"] = \"xxxxxxxxxxxxxxxxx\"\n",
+    "os.environ[\"POLYTOPE_PASSWORD\"] = \"jxcmecNjioyMitMk\"\n",
```
Comment (ninaburg) [inline notebooks/4_Location_and_TimeSeries_Access.ipynb:33]:
```suggestion
    "os.environ[\"POLYTOPE_PASSWORD\"] = \"xxxxxxxxxxxxxxxxx\"\n",
```

**Comment by cfkanesan** [comment]
@ninaburg I deleted the comments to avoid showing the change but it should be gone now.

**Review by ninaburg — APPROVED** [review]
The changes are fine for me and all notebooks could be tested and are working.

**Pull request merged**